torch/lib/THCS/generic/THCSTensor.cu

```diff
@@ -67,7 +64,8 @@ THCSTensor *THCSTensor_(newCoalesce)(THCState *state, THCSTensor *self) {
 
   cudaStream_t stream = THCState_getCurrentStream(state);
 
-  THCIndexTensor *indices1D = THCSTensor_(newFlattenedIndices)(state, self);
+  // indices will be modified, so we have to clone or use new storage here
```

Comment:
It doesn't take much more typing to say "modified by Thrust", which will be helpful for anyone else who ever wants to read this code ;)